monitoring-guidelines.md

```diff
@@ -55,13 +55,19 @@ When creating a KubeVirt alert rule, please follow the [OpenShift Alerting Consi
 In addition to the [OpeShift Style Guide](https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/alerting-consistency.md#style-guide) the KubeVirt alerts MUST include:
 1. `kubernetes_operator_part_of` label indicating the operator name. Value should be set to `kubevirt`.
 2. `kubernetes_operator_component` label indicating the value of the sub operator name.
-
+3. `operator_health_impact` label indicating how the alert impacts the operator's functionality. 
+This label differs from `severity`, as severity can be tied to workload issues (e.g VMIs), while  `operator_health_impact` refers only to the impact the issue has on the operator itself.
+
+   Valid values for this labels are:
+   - `critical` - For alerts that impact the operator's functionality badly, and an action must be taken immediately in order to solve it.
+   - `warning` - For alerts that soon might impact the operator's functionality badly, and the user should be warned in order to solve the issue.
+   - `none` - For alerts that don't affect the operator's functionality, and are related to workloads rather than the operator itself.
```

Comment:
>`none` - For alerts that don't affect the operator's functionality, and are related to workloads rather than the operator itself.

`none` -  For alerts that don't impact the operator's functionality.